lib/modules/default.nix

```diff
@@ -4,4 +4,5 @@ rec {
   keyboard = import ./keyboard.nix { inherit lib; };
   neovim = import ./neovim.nix { inherit lib; };
   themes = import ./themes.nix { inherit lib neovim zsh; };
```

Comment:
Commented so that gets removed from #31, let's be sure it still stays here. This PR is dependent of #31 so we can just not care.

Reply:
Yep I merged every change up the chain of PRs so it's all good. Thanks!